config/initializers/paperclip.rb

```diff
@@ -30,6 +30,43 @@ def nil?
   end
 end
 
+module PaperclipBlurhash
+  def blurhash
+    return unless instance.respond_to?(:"#{name}_meta") && instance_read(:meta)
+
+    if (meta = meta_decode(instance_read(:meta)))
```

Comment:
This _kinda_ bugs me 😆 . I have always hated inline assignments inside of `if` statements.

Reply:
Me too, but it's what they use in the code I pasted so 🤷

Reply:
Ok, at-least I ain't crazy for thinking that.